soc/nordic/nrf52/Kconfig

```diff
@@ -31,6 +31,10 @@ config SOC_NRF52840
 
 if SOC_SERIES_NRF52X
 
+config SOC_NORDIC_BSP_PATH
```

Comment:
what about `SOC_NORDIC_BSP_{DIR,NAME}` rather than `_PATH` ? `_PATH` could mean complete path, which could be confusing if it is later used to build actual paths like `set(MDK_DIR ${NRFX_DIR}/bsp/${CONFIG_SOC_NORDIC_BSP_PATH}/mdk)`

Reply:
Changed to `SOC_NORDIC_BSP_NAME`